examples/terraform-submodules/eks/module.tf

```diff
@@ -0,0 +1,70 @@
+// Copyright 2019 Google LLC All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+// Run:
+//  terraform apply [-var agones_version="1.0.0"]
+
+// Install latest version of agones
+variable "agones_version" {
+  default = "1.0.0"
+}
+variable "cluster_name" {
+  default = "agones-cluster"
+}
+
+variable "region" {
+  default = "us-west-2"
+}
+
+provider "aws" {
+  version = "~> 2.8"
+
+  region  = var.region
+}
+
+variable "machine_type" { default = "t2.large" }
+
+module "eks_cluster" {
+  source = "git::https://github.com/googleforgames/agones.git//install/terraform/modules/eks/?ref=master"
+
+  machine_type = "${var.machine_type}"
+  cluster_name = "${var.cluster_name}"
+}
+
+data "aws_eks_cluster_auth" "example" {
+  name = "${var.cluster_name}"
+}
+
+// TODO(alekser): Add Helm submodule
```

Comment:
Curious - what's this for?

Reply:
If I add Helm submodule right now , `terraform destroy` would not succeed. When EKS Terraform provider would be fixed, this section would be uncommented. I cannot estimate when it would happen.

Reply:
Got it :+1: can you add that note below your TODO, so we have some context? Just in case someone else ends up working on this later on?